addons/utm/models/utm_campaign.py

```diff
@@ -42,6 +42,8 @@ def create(self, vals_list):
         for vals in vals_list:
             if not vals.get('title') and vals.get('name'):
                 vals['title'] = vals['name']
+            elif vals.get('title') and not vals.get('name'):
+                vals['name'] = vals['title']
```

Comment:
@std-odoo isn't that the job of the precompute actually?

I think the issue here is that we force a False value in the "name" key of the dict, meaning the precompute does not do its job properly.

I tried changing it this way:
```py
for vals, new_name in zip(vals_list, new_names):
    if new_name:
        vals['name'] = new_name
```

Seems to do what we want when testing locally:

![image](https://user-images.githubusercontent.com/41282536/213226137-120cf6b2-af75-4b93-991a-0b0cc8bb87be.png)

Maybe I'm missing something, this is rather complicated :smile:

Reply:
Right, changed it to your approach, thanks :)